commands/commands.go

```diff
@@ -74,16 +75,20 @@ func createCancellableContext() context.Context {
 	return ctx
 }
 
-func suggestSettingBuilder(logger *logging.Logger, client PackClient) {
-	logger.Info("Please select a default builder with:\n")
-	logger.Info("\tpack set-default-builder <builder image>\n")
+func suggestSettingBuilder(logger logging.LoggerWithWriter, client PackClient) {
+	out := logger.Writer()
```

Comment:
`logger.Infof` seems simpler than this. Is there something I'm missing?

Reply:
These are normal informational messages that shouldn't be prefixed with INFO/DEBUG/ERROR and are always displayed no matter the desired log level, so we just output the text as formatted.